### PR TITLE
Fix phpDoc in Db

### PR DESCRIPTION
**Pull request by ksaandev**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix phpDoc in Db
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | No behavior is modified

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22183)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: classes/db/Db.php [hunk 1]
```diff
@@ -776,8 +776,9 @@ public function displayError($sql = false)
     /**
      * Sanitize data which will be injected into SQL query.
      *
-     * @param string $string SQL data which will be injected into SQL query
-     * @param bool $html_ok Does data contain HTML code ? (optional)
+     * @param string $string  SQL data which will be injected into SQL query
+     * @param bool   $html_ok Does data contain HTML code ? (optional)
+     * @param bool   $bq_sql  Escape backticks
```
Comment (Progi1984) [inline classes/db/Db.php]:
```suggestion
     * @param bool $html_ok Does data contain HTML code ? (optional)
     * @param bool $bq_sql  Escape backticks
```

**Review by PululuK — DISMISSED** [review]

File: classes/db/Db.php [hunk 2]
```diff
@@ -776,8 +776,9 @@ public function displayError($sql = false)
     /**
      * Sanitize data which will be injected into SQL query.
      *
-     * @param string $string SQL data which will be injected into SQL query
+     * @param string $string  SQL data which will be injected into SQL query
```
Comment (matks) [inline classes/db/Db.php]:
```suggestion
     * @param string $string SQL data which will be injected into SQL query
```

File: classes/db/Db.php [hunk 3]
```diff
@@ -776,8 +776,9 @@ public function displayError($sql = false)
...
      * @param bool $html_ok Does data contain HTML code ? (optional)
+     * @param bool $bq_sql  Escape backticks
```
Comment (matks) [inline classes/db/Db.php]:
```suggestion
     * @param bool $bq_sql Escape backticks
```

**Review by matks — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Comment by PululuK** [comment]
Thanks @ksaandev

**Comment by ksaandev** [comment]
Thank you for the fast merge you guys are amazing!

**Pull request merged**